### ISSUE TITLE
Am reception bandwidth?

### ISSUE THREAD

**Issue by ea3grn** [Tunas1337/UV-K5-Modded-Firmwares#7]
Hello,

it seems, according to some web comments, that the AM reception in this firmware is set to a bandwidth of 8.33 Khz. Is it possible to try to set for the aeronautical band that the AM bandwidth is 25 Khz? I have not been able to find the chipset command guide, but maybe it is possible to make the change in the firmware to see if this improves the AM reception quality (right now pretty bad).

If it is possible for you, would you be so kind to see if it is possible to implement this change?

**Comment by Tunas1337** [comment]
It's on the list of stuff to look at. Someone sent something promising, I'll check it out.

**Comment by ea3grn** [comment]
If I can be of any help, I am at your disposal.

**Comment by pawol** [comment]
I do not think so. It is possible to change channel width on AM between 6.25 and 12.5 kHz and the difference is easy to hear. 8.33kHz is only used in context of frequency raster and nothing more.

**Comment by Tunas1337** [comment]
So, wide/narrow works for AM?

**Comment by pawol** [comment]
sure. Do not mix bandwidth 6.25/12,5 kHz (depends on nad plan) with frequency raster 2.5, 5,0, 6,25, 12,5, 25  and 8.33 kHz
I have noticed that AM on 6.25 bandwidth is less sensitive and hissing then 12,5 bandwidth. Same setting are for FM and there ones are called NFM or FM (6.25 or 12.5 kHz)

**Comment by pawol** [comment]
Just set some airband frequency, disable squelch (menu 0 --> 0) and change band width (menu 10 --> wide or narrow)

**Comment by ea3grn** [comment]
I can confirm there is no improvement between NARROW and WIDE mode in AM. In both modes the audio is distorted.

**Comment by pawol** [comment]
Sure, because it makes an influnce on audio band pass only. IMHO there is problem with gain. It seems to be possible to control one by DSP

**Comment by SteefW** [comment]
> Sure, because it makes an influnce on audio band pass only. IMHO there is problem with gain control or DSP signal processing.  It seems to be possible to control one by DSP. DSP cuts hight AM peaks of amplitude like guitar fuzz effect.

That's just it i guess. Sounds like distorted AGC as well. It seems that especially with strong signals it looks like the problem is bigger. 
Thanks for the attencion about this prob. Did not try UHF AM though yet.

**Comment by Sthrenor** [comment]
According to SDR airband reception bandwidth is only 2khz for air band. Maybe lowering it is the solution. Just give it a try if posible.

**Comment by pawol** [comment]
Nonsense. 2kHz of channel width results with 1kHz voice band pass only. Standard AM with voice up to 3kHz (normal for telecommunication purposes) have to have at least channel band width 6kHz

**Comment by Sthrenor** [comment]
If you listen to aircraft emisions with an SDR you will see that the bandwidth is 2khz. I will make a video about it and will link it here. I suspect that quansheng airband bandwidth is 12,50 or 25khz as is FM transmissions and that is why we get all that background noise on airband. If is testable....

**Comment by pawol** [comment]
Understand one thing - 2 kHz of channel band width is again the AM modulation theory. max voice frequency *2 gaves us channel band with. Have tou ever heard chat on 1KHz wide channel?

**Comment by pawol** [comment]
AM channel can be wider then expected but the selectivity will be smaller and the sensitivity will be higher.

**Comment by Sthrenor** [comment]
I listen to airband with my HRD-747 set at AM 2khz bandwidth perfectly and with SDR. With the quansheng I cant. It enters tons of background noise and I thought that could be these. But if you say it goes against AM modulation theory, don't try it... It was a suggestion. Sorry for the inconvenience.

**Comment by Sthrenor** [comment]
Is at least posible to set the AM bandwidth with the theorical minimun bandwidth for voice of 6khz?

**Comment by gg33fr** [comment]
Hello, I found this video.
Replacing a CMS component
audio Hi-Cut (1nF capacitor)
If anyone has the material to try.
THANKS

https://www.youtube.com/watch?v=j4ccuYOg2NU

**Comment by pawol** [comment]
Nothing common with AM reception

**Comment by ericek111** [comment]
@marusc1
Even though the frequencies seemingly end with a 0 or 5, in reality, they're transmitting on different frequencies. European Union has implemented the requirement for 8.33 kHz-capable radios above FL195 in 2018. Refer to this image:

![](https://i.imgur.com/ghUHI2K.png) 

One advantage of the 25 kHz frequencies is that they allow to extend the range using CLIMAX (+/- 2.5 kHz carrier offset).
